Header header
float32 tx  # target x position (pixels)
float32 ty  # target y position (pixels)
float32 thor  # target width (pixels)
float32 tvert  # target height (pixels)
